packages/marko-web-identity-x/browser/access.vue

```diff
@@ -229,6 +234,26 @@ export default {
       return this.user && this.user.email;
     },
 
+    /**
+     *
+     */
+    redirect() {
+      if (this.content.siteContext && this.content.siteContext.url) {
+        return this.content.siteContext.url;
+      }
+      if (this.content) {
+        return this.content.id;
```

Comment:
I _think_ this should be prefixed with a slash. Can you confirm what the login/profile components do with the redirect prop? Based on the other two values though, this probably should include the leading slash.

Reply:
@solocommand Based on what im reading the login component takes this and uses it when set or generates its own off of the window.locations... ${pathname}${search}${hash} I was setting this more as an insurance.